react-frontend/src/components/Graph2D.js

```diff
@@ -32,6 +49,9 @@ class Graph2D extends React.Component {
         currentZoomLevel: 1, // storing the current zoom level
         // !TODO: to be used for re-setting the state

```

Comment:
What's the meaning of this comment? I think it's not a ToDo, is it?

Reply:
No. It is no longer a todo as we have moved the filtering logic to server-side. Removing it.